groups/infrastructure/main.tf

```diff
@@ -0,0 +1,94 @@
+terraform {
+  required_version = "~> 1.3"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.54.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = "~> 3.18.0"
+    }
+  }
+}
+
+provider "aws" {
+  region  = var.aws_region
+}
+
+terraform {
+  backend "s3" {}
+}
+
+module "file_transfer_alb" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/application_load_balancer?ref=1.0.296"
+  count = "${var.file_transfer_create_alb == 1 ? 1 : 0}"
+
+  environment         = var.environment
+  service             = "file-transfer"
+  ssl_certificate_arn = data.aws_acm_certificate.cert.arn
+  subnet_ids          = values(local.routing_subnet_ids)
+  vpc_id              = data.aws_vpc.vpc.id
+  idle_timeout        = 1200
+  route53_domain_name     = var.cert_domain
+
+  create_security_group  = true
+  internal               = var.alb_internal
+  ingress_cidrs          = ["0.0.0.0/0"]
+  service_configuration  = {
+    listener_config = {
+        default_action_type = "fixed-response"
+        protocol            = "HTTPS"
+        port                = 443
+      }
+  }
+}
+
+
+module "secure_file_transfer_alb" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/application_load_balancer?ref=1.0.296"
+  count = "${var.file_transfer_create_alb == 1 ? 1 : 0}"
+
+  environment         = var.environment
+  service             = "secure-file-transfer"
+  ssl_certificate_arn = data.aws_acm_certificate.cert.arn
+  subnet_ids          = values(local.routing_subnet_ids)
+  vpc_id              = data.aws_vpc.vpc.id
+  idle_timeout        = 1200
+  route53_domain_name     = var.cert_domain
+
+  create_security_group  = true
+  internal               = var.alb_internal
+  ingress_cidrs          = ["0.0.0.0/0"]
+  service_configuration  = {
+    listener_config = {
```

Comment:
Should the secure return 404 ?

Reply:
Why should we return 404? Any specific security reason?

Reply:
From **Ben:** 
I think both should probably be a 404. I doubt it actually is happening anywhere but it just prevents the possibility of a service somewhere treating a 2XX response as a success when strictly this was a failed upload/download

[10:05](https://companieshouse.slack.com/archives/C084MM8S96V/p1735034753415619)
And some response just to say route not found

Reply:
This is fixed now. Commit is pushed.